subprojects/docs/src/docs/userguide/buildEnvironment.xml

```diff
@@ -79,6 +79,12 @@
                 </listitem>
             </varlistentry>
             <varlistentry>
+                <term><literal>org.gradle.log.level</literal></term>
```

Comment:
Please pay special attention to what you think this should be.  I'm not sure it shouldn't be `org.gradle.logging.level` or maybe `org.gradle.loglevel`.